axelrod/tests/strategies/test_titfortat.py

```diff
@@ -114,6 +114,34 @@ def test_strategy(self):
         self.versus_test(opponent, expected_actions=actions)
 
 
+class TestDynamicTwoTitsForTat(TestPlayer):
+
+    name = 'Dynamic Two Tits For Tat'
+    player = axelrod.DynamicTwoTitsForTat
+    expected_classifier = {
+        'memory_depth': 2,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        self.first_play_test(C)
+        self.second_play_test(rCC=C, rCD=D, rDC=C, rDD=D)
+
+        # Will cooperate if opponent cooperates/.
+        actions = [(C, C), (C, C), (C, C), (C, C), (C, C)]
+        self.versus_test(axelrod.Cooperator(), expected_actions=actions)
+        # Will defect twice when last turn of opponent was defection.
+        opponent = axelrod.MockPlayer(actions=[D, C, C, D, C])
+        actions = [(C, D), (D, C), (D, C), (C, D), (D, C)]
+        self.versus_test(opponent, expected_actions=actions)
```

Comment:
> I'm not sure I follow. I choose a different seed, and then do I repeat the same test?

That's right with different actions.

To illustrate:

```python
>>> p = 0.5 # for example opponent.history = [C, C, D, D]
>>> axl.seed(0)
>>> axl.random_choice(p)
'D'
>>> axl.seed(1)
>>> axl.random_choice(p)
'C'
```

So with different seeds the random choice gives different results.

So in your case you'd have something like:

```python
actions = [..., (D, D)]
self.versus_test(opponent=opponent, expected_actions=actions, seed=1)
actions = [..., (C, D)]  # Different actions
self.versus_test(opponent=opponent, expected_actions=actions, seed=0)  # Different seed
```

This just shows that the strategy does act randomly.